.github/CODEOWNERS

```
@@ -642,6 +642,7 @@ peps/pep-0760.rst  @pablogsal @brettcannon
 peps/pep-0761.rst  @sethmlarson @hugovk
 peps/pep-0762.rst  @pablogsal @ambv @lysnikolaou @emilyemorehouse
 peps/pep-0763.rst  @dstufft
+peps/pep-0764.rst  @Viicos @erictraut
```

CODEOWNERS can only list committers.
```suggestion
# peps/pep-0764.rst
```

---

You can add me instead (I'm not the sponsor but I can redirect any PRs to the two of you).
```suggestion
peps/pep-0764.rst  @JelleZijlstra
```